### ISSUE TITLE
If keyword has a long list of arguments some of them are moved into keyword itself

### ISSUE THREAD

**Issue by povar81** [robotframework/RIDE#2727]
RIDE v2.0.8.1 running on Python 3.7.4. 
my keyword in text edit:
```
aar_i_3 subscr_id ipv6
    [Arguments]    ${msisdn1}    ${subscrtype1}    ${msisdn2}    ${subscrtype2}    ${msisdn3}    ${subscrtype3}    ${req_access_info}    ${af_req_data}    ${appid}    ${apn}    ${Media-Type}    ${Flow-Status}    ${Flow-Usage}    ${sessionid}    ${MRBULd}    ${MRBDL}
    ...    ${MinRBU}    ${MinRBD}    ${ip_v6}
 ```   
 in grid editor last 3 arguments are not in arguments but in the keyword:
 
![image](https://github.com/robotframework/RIDE/assets/7736848/43b04250-7822-4863-8a2e-1390a98d98a1)

and when this keyword is called from a case the last cell on first line is shown in red as an error:
![image](https://github.com/robotframework/RIDE/assets/7736848/b81e3e85-a1cd-4b3d-b331-023cdff4cdbd)
and second line is shown as unknown:
![image](https://github.com/robotframework/RIDE/assets/7736848/56f7e907-aa8f-42ac-911b-4b66d97f22ae)

the same keyword on 
RIDE 1.7.4.2 running on Python 3.7.4. :
![image](https://github.com/robotframework/RIDE/assets/7736848/b57a36d4-a792-447e-b624-884f21a23843)

and how it's shown when called from a case:
![image](https://github.com/robotframework/RIDE/assets/7736848/d3adcda6-41f7-444a-9d7b-87d63d32eab7)

**Comment by yushiwolele** [comment]
I have the same issue in version 2.1.
![123](https://github.com/user-attachments/assets/1deff64c-fd66-4b9f-89e6-a0a929b64629)
 RIDE 2.1,wxPython 4.2.1

**Comment by HelioGuilherme66** [comment]
Fixed in `develop` future 2.1.1.
Colorization of arguments will not be fixed.
